sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -609,7 +611,8 @@ def __init__(
       coder=None,
       use_standard_sql=False,
       flatten_results=True,
-      kms_key=None):
+      kms_key=None,
+      backwards_compatible_data_format=None):
```

Comment:
1. Can we use a bool as a default value instead of `None`?
2. Can we use rename this flag to something that describes the course of action, for example: `use_json_data_representation=False` - you can come up with a better name.

Reply:
1. Done.
2. I've called it `use_json_exports` - does that help?